awx/main/scheduler/dag_simple.py

```diff
@@ -1,24 +1,56 @@
-
```

Comment:
I'm getting confused about this -

I thought that DAG simple was for task manager options, but this file is now full of references to workflow nodes. I thought those should only be relevant for DAG workflow.